src/ale_interface.cpp

```diff
@@ -111,7 +141,32 @@ ALEInterface::ALEInterface(bool display_screen) {
   this->setBool("display_screen", display_screen);
 }
 
-ALEInterface::~ALEInterface() {}
+ALEInterface::~ALEInterface() {
+}
+
+bool is_illegal(char c) {
+  return !(std::isalnum(c) || c == '_');
+}
+
+string ALEInterface::find_rom(const string& md5) {
+  // TODO these values should either come from an external text file or be part of the RomSettings subclasses
+  // TODO building a dictionary would be more elegant, but overkill for so few strings
+  // TODO this can be done much more elegantly with C++11
+  std::ifstream ss("md5.txt");
```

Comment:
the comment is partly outdated; originally I just hardcoded the strings in the .cpp file.